extenders/Message.js

```diff
@@ -19,11 +19,11 @@ module.exports = Structures.extend("Message", Message => class extends Message {
     return this.channel.send(`${this.author} \`|${emoji}|\` ${content}`, Object.assign({}, options,  { embed }));
   }
 
-  async awaitReply(question, filter, limit = 60000, embed) {
-    await this.channel.send(question, embed);
-    return this.channel.awaitMessages(filter, { max: 1, time: limit, errors: ["time"] })
+  async awaitReply(question, filter, limit = 60000) {
+    const msg = await this.channel.send(question);
+    return [await this.channel.awaitMessages(filter, { max: 1, time: limit, errors: ["time"] })
```

Comment:
you should not be returning it in as a array... it should be ```js
    const msg = await this.channel.send(question);
    return this.channel.awaitMessages(filter, { max: 1, time: limit, errors: ["time"] })
      .then(collected => { msg, response: collected.first().content })
      .catch(() => false);
``` or something close to that but sending the array is a very bad practice and confusing for people to read where awaitReply is used

Reply:
What difference in the code does it make? It's still going to need changes to other pieces.
In this case you're going to have to do reply.msg or reply.response.